### PR TITLE
Enterprise license FAQ improvements

### PR DESCRIPTION
**Pull request by jkirschner-hashicorp**
Improvements include:
- Clarifying license expiration behavior
- Adding a license renewal Q+A
- Adding obtaining a copy of your license as an existing customer Q+A
- Clarify evaluation license changes

[Preview](https://consul-r4rx5u4zh-hashicorp.vercel.app/docs/enterprise/license/faq)

To-Do:
- [ ] Get confirmation on accuracy of "clarify evaluation license changes" (last commit)

### CODE REVIEW

File: website/content/docs/enterprise/license/faq.mdx [hunk 1]
```diff
@@ -56,15 +56,18 @@ The in-storage license feature will not be supported starting with Consul Enterp
...
 ## Q: Is there a grace period when licenses expire?
 
-A license includes an `expiration_date` and a `termination_date`. An enterprise binary will cease to function once the `termination_date` has passed.
-Licenses will now have a 24 hrs grace period. The grace period is the time between license expiry until the time it terminates.
-As Consul Enterprise approaches the expiration date, warnings will be issued in the system logs.
+**Yes—Consul will continue normal operation after license *expiration*** as defined in
+[`expiration_time`](/api-docs/operator/license#getting-the-consul-license).
+As license expiration is approached or passed, Consul will issue warnings in the system logs.
+
+Consul will only cease operation after license *termination*, which occurs 10 years
```
Comment (karl-cardenas-coding) [inline website/content/docs/enterprise/license/faq.mdx:68]: 10 years is a long time. So what's stopping anyone from not renewing the license and have Consul for another 10 yrs?

File: website/content/docs/enterprise/license/faq.mdx [hunk 2]
```diff
@@ -46,19 +46,20 @@ The list below is a great starting point for learning more about the license cha
...
+This impacts customers that have an enterprise binary (trial or non-trial licenses) downloaded from https://releases.hashicorp.com.
 The license changes do not impact customers with the baked-in licensed binaries. In a later release of Consul Enterprise, baked-in binaries will be deprecated.
 
 ## Q: What is the product behavior change introduced by the licensing changes?
 
 Starting with Consul Enterprise 1.10.0, a valid license is required on-disk (auto-loading) or as an environment variable for Consul Enterprise to successfully boot-up.
 The in-storage license feature will not be supported starting with Consul Enterprise 1.10.0+ent. All Consul Enterprise clusters using 1.10.0+ent must ensure that there is a valid license on-disk (auto-loaded) or as an environment variable.
 
-## Q: What is the impact on evaluation licenses due to this change?
+## Q: What is the impact on trial licenses due to this change? ((#q-what-is-the-impact-on-evaluation-licenses-due-to-this-change))
```
Comment (jkirschner-hashicorp) [inline website/content/docs/enterprise/license/faq.mdx:57]: Added anchor link alias for backwards compatibility with old heading

**Review by david-yu — APPROVED** [review]

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/598542.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 9c74f140616808b1cc79d68402fc76f20ce2084d onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 9c74f140616808b1cc79d68402fc76f20ce2084d onto `release/1.11.x` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 9c74f140616808b1cc79d68402fc76f20ce2084d onto `release/1.10.x` succeeded!

**Pull request merged**